src/components/ClaimENSPage/ClaimENSPage.tsx

```diff
@@ -84,9 +89,10 @@ export default class ClaimENSPage extends React.PureComponent<Props, State> {
   }
 
   render() {
-    const { onBack } = this.props
+    const { onBack, ensList } = this.props
     const { name, isLoading } = this.state
     const isValid = isNameValid(name)
+    const isRepeated = isNameRepeated(name, ensList)
```

Comment:
Oh you're right!!! we need to avoid repeated names from the entire name registry!!! ... I think is not possible doing it every time that the user press a key. We'll need to check it maybe when the Fat Finger Protection modal show up. If is repeated, then you show the error in the interface without triggering the modal.

Reply:
I'll keep this as is but adding some comments in the code.